src/ResourceOperations/ResourceHolder.cs

```diff
@@ -144,6 +144,7 @@ private void UpdateFile(string filename, string valueColumnId, bool skipNontrans
                     {
                         originalMetadatas.Add((string)metadataEnumerator.Key, metadataEnumerator.Value);
                     }
+                    reader.Close();

```

Comment:
`reader.Close();` is not necessary, it's already called by `using (var reader...)` when exiting the scope.

Reply:
Yes, really it isn't necesary. I didn't see the using, i supose.